ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-rollback-stable.yaml

```diff
@@ -26,6 +26,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: 4.17.0-ec.3
+  source:
```

Comment:
stable ci does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks